src/chart/rendering/drawlabel.cpp

```diff
@@ -63,15 +49,12 @@ void DrawLabel::draw(Gfx::ICanvas &canvas,
 	    copyRect.transform
 	    * Geom::AffineTransform(alignRect.bottomLeft());
 
-	if (options.alpha) {
-		if (auto textColor = *style.color * *options.alpha;
-		    options.gradient)
-			options.gradient(canvas,
-			    textTransform.inverse(),
-			    textColor);
-		else
-			canvas.setBrushColor(textColor);
-	}
+	if (auto &&textColor = options.colorTransform(*style.color);
```

Comment:
Since we removed the background, i'm not sure if it still makes sense to setup the brush here. We could setup it outside, then call DrawLabel